experiment/dispatcher.py

```diff
@@ -133,18 +133,16 @@ def dispatcher_main():
                                              args=(experiment.config,))
     scheduler_loop_thread.start()
 
-    max_total_time = experiment.config['max_total_time']
-    measurer_loop_process = multiprocessing.Process(
-        target=measurer.measure_loop,
-        args=(experiment.experiment_name, max_total_time))
+    measurer_main_process = multiprocessing.Process(
```

Comment:
Is this code being added for the first time? I feel like I've seen this before?

Reply:
Yeah, It is the rebased version of #565.